cordapp-cpk/src/test/resources/cordapp-cpb/build.gradle

```diff
@@ -0,0 +1,38 @@
+plugins {
+    id 'net.corda.plugins.cordapp-cpb'
+    id 'org.jetbrains.kotlin.jvm'
+}
+
+apply from: 'javaTarget.gradle'
+apply from: 'kotlin.gradle'
+
+allprojects {
+    group = 'com.example'
+    version = cordapp_version
+    repositories {
+        maven {
+            name = 'test-repository'
+            url = getProperty("maven.repository.dir")
+        }
+        mavenCentral()
+    }
+}
+
+version = "1.0"
+
+cordapp {
+    targetPlatformVersion = platform_version.toInteger()
+
+    contract {
+        name = 'CorDapp CPB test'
+        versionId = cordapp_contract_version.toInteger()
+        licence = 'Test-Licence'
+        vendor = 'R3'
+    }
+}
+
+dependencies {
+    cordapp group: group, name: "external-cordapp", version: cordapp_version
+    cordapp project('project-dependency')
+}
+
```

Comment:
And ask Gradle to copy all dependent CPKs somewhere more convenient. The `copyCPKs` task will be executed as a dependency of the `assemble` task:
```gradle
configurations {
    cpks
}

dependencies {
    cpks project(path: 'project-dependency', configuration: 'cordaCPK')
    cpks "com.example:corda-platform-cordapp:$cordapp_version:cordapp@cpk"
    cpks "com.example:external-cordapp:$cordapp_version:cordapp@cpk"
    cpks "com.example:external-cordapp-transitive-dependency:$cordapp_version:cordapp@cpk"
}

def cpkDir = file("$buildDir/cpks")
def copyCPKs = tasks.register('copyCPKs', Copy) {
    from configurations.cpks
    into cpkDir
}

artifacts {
    archives(file: cpkDir) {
        builtBy copyCPKs
    }
}
```